### ISSUE TITLE
Help?

### ISSUE THREAD

**Issue by LiteNiteFrack** [PoESkillTree/PoESkillTree#550]
I had an old version working for me yesterday. I love it! Better that PoE Planner and Path of Building. WarfortheAtlas-3.0.0-alpha.2. I tried to use it today. No go. Crashed. Did not keep the log.

Downloaded and tried to run the latest, PoESkillTree-ConquerorsoftheAtlas-3.0.0-alpha.8 and I got the following crash log and am including the main  log as well. I install  the program on my D drive and I will keep the build folder there as well. I do not mess with the stuff that is installed  on the C drive. Except to get the logs. Any help?

Thank you

[2020-01-9 ConquerorsoftheAtlas-3.0.0-alpha.8 main.log](https://github.com/PoESkillTree/PoESkillTree/files/4043564/2020-01-9.ConquerorsoftheAtlas-3.0.0-alpha.8.main.log)
[2020-01-9 ConquerorsoftheAtlas-3.0.0-alpha.8crash.log](https://github.com/PoESkillTree/PoESkillTree/files/4043565/2020-01-9.ConquerorsoftheAtlas-3.0.0-alpha.8crash.log)

**Comment by brather1ng** [comment]
Looks like your passive tree data is corrupt. Try deleting `Data\SkillTree.txt` (the folder is next to the `logs` folder).

**Comment by LiteNiteFrack** [comment]
Downloaded PoESkillTree-ConquerorsoftheAtlas-3.0.0-alpha.8.   I installed the program on my D drive and  will keep the build folder there as well. I did not mess with the stuff that is installed on the C drive. Tried to run it. Got a crash error. Went and deleted the SkillTree.txt file. Ran again. Crash.
I am including the crash log and the main log.

[main.log](https://github.com/PoESkillTree/PoESkillTree/files/4049038/main.log)
[crash.log](https://github.com/PoESkillTree/PoESkillTree/files/4049039/crash.log)

[C Drive main.log](https://github.com/PoESkillTree/PoESkillTree/files/4049051/C.Drive.main.log)
[C Drive crash.log](https://github.com/PoESkillTree/PoESkillTree/files/4049052/C.Drive.crash.log)
I tried letting everything be installed on my C Drive. I am including the crash and main log 
from that attempt as well. Oh well. Maybe it is just one of those things.

**Comment by brather1ng** [comment]
Looks like it's not `SkillTree.txt` but `Opts.txt`. When I delete that file or redownload live skill tree data, I get the same error as you.

Try replacing your `Data\Opts.txt` with this [Opts.txt](https://github.com/PoESkillTree/PoESkillTree/files/4049194/Opts.txt). That's the one included in the alpha.8 release. What I don't know is why it's not being replaced when you re-install the program.

I'll look into what GGG changed in their data leading to these problems later.

**Comment by LiteNiteFrack** [comment]
Stared from scratch. Deleted everything. Downloaded PoESkillTree-ConquerorsoftheAtlas-3.0.0-alpha.8 to my D drive, keep the build folder there as well. Let the stuff install on C drive. Ran it. Crash. Replaced the Opts.txt file. Ran it. It crashed. I am stubborn so I ran  it again with my fingers crossed. Crashed again and I could faintly see the skill tree in  the background. It was in the Initialization stage.

Here are the crash log and  the main log.
[crash.log](https://github.com/PoESkillTree/PoESkillTree/files/4054358/crash.log)
[main.log](https://github.com/PoESkillTree/PoESkillTree/files/4054359/main.log)
